.vscode/settings.json

```diff
@@ -0,0 +1,3 @@
+{
+    "python.pythonPath": "venv/bin/python"
+}
```

Comment:
Can you please not commit this file. Can you update the .gitignore to ensure this doesn't get committed again?